File: app/src/main/java/eu/toma/dev/playground/warcraft/model/boss/Npc.java
package eu.toma.dev.playground.warcraft.model.boss;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Npc
{
    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("urlSlug")
    @Expose
    public String urlSlug;
}

File: app/src/main/kotlin/warcraft/model/boss/Npc.kt
package warcraft.model.boss

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Npc {
    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("name")
    @Expose
    var name: String
    @SerializedName("urlSlug")
    @Expose
    var urlSlug: String
}
